detailed_docs/TRS/RC/OnRCStatus.md

```diff
@@ -0,0 +1,163 @@
+## Functional Requirements  
+
+1.
+In case
+- RC functionality is allowed on HMI  
+- all RC modules are free
+
+AND RC application sends ButtonPress or SetInteriorVehicleData request 
+
+SDL must  
+send OnRCStatus notification with RC modules in 'freeModules' and 'allocatedModules' parameters to HMI  
+send OnRCStatus notification with 'allowed = true', 'freeModules' and 'allocatedModules' parameters to mobile application 
+
+2.
+In case   
+- RC functionality is disallowed on HMI  
+- all RC modules are free
+
+AND RC application sends ButtonPress or SetInteriorVehicleData request 
```

Comment:
the request to ButtonPress or SetInteriorVehicleData shall be reject, no need to send  OnRCStatus for the requests

Reply:
@yang1070, thank you
please, see updates https://github.com/smartdevicelink/sdl_requirements/pull/89/commits/9a23cb27fb2e940fd3601f71687d8665ce8618e6